src/test/java/seedu/address/logic/parser/ViewTasksCommandParserTest.java

```diff
@@ -52,14 +52,14 @@ public void parse_validTaskDescArgs_returnsViewTasksCommand() {
         assertParseSuccess(parser, " " + PREFIX_TASK_DESCRIPTION + "prs book", expectedCommand);
 
         // multiple whitespaces between keywords
-        assertParseSuccess(parser, " " + PREFIX_TASK_DESCRIPTION + "prs \n \t book", expectedCommand);
+        assertParseSuccess(parser, " " + PREFIX_TASK_DESCRIPTION + "prs \t book", expectedCommand);
     }
 
     @Test
     public void parse_validTaskProgressArgs_returnsViewTasksCommand() {
         // no leading and trailing whitespaces
         ViewTasksCommand expectedCommand =
-                new ViewTasksCommand(new TaskProgressContainsKeywordsPredicate(Arrays.asList("done", "pending")));
+                new ViewTasksCommand(new TaskProgressContainsKeywordsPredicate(Arrays.asList("DONE", "PENDING")));
```

Comment:
This suppose to test for case sensitivity?

Reply:
same for line 73

Reply:
the enum.name() returns the capital form so the equality check will fail if its not capitalised